### PR TITLE
Add sizes for tags

### PR DESCRIPTION
**Pull request by collincchoy**
image

### CODE REVIEW

**Comment by pleek91** [comment]
@brandonreid

**Review by brandonreid — APPROVED** [review]
One comment, otherwise g2g!

File: src/components/Tag/PTag.vue [hunk 1]
```diff
@@ -23,12 +23,13 @@
...
   import { TagValue, normalize } from '@/types/tag'
 
-  const props = defineProps<{
+  const props = withDefaults(defineProps<{
     icon?: Icon,
     label?: string,
     value?: string | TagValue,
     dismissible?: boolean,
-  }>()
+    size?: 'xs' | 'sm',
```
Comment (brandonreid) [inline src/components/Tag/PTag.vue:31]: To be a little more semantically prescriptive, maybe the terms here would be better suited as `'sm' | 'default'`? Or, like buttons, we ended up refactoring to just the default and a bool prop called `small`.

Comment (brandonreid) [reply src/components/Tag/PTag.vue:31]: I'm like 98.6% sure we'll never add a `large` for tags

Comment (collincchoy) [reply src/components/Tag/PTag.vue:31]: 👍 will shoot for consistency. size -> `small?: boolean`

**Pull request merged**